sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -270,4 +274,18 @@ Object templateFromPromptRegistryByScenario(
     }
     return response.getContent();
   }
+
+  @GetMapping("/localPromptTemplate")
+  @Nonnull
+  Object localPromptTemplate(@RequestParam(value = "format", required = false) final String format)
+      throws IOException {
+    final var resource = resourceLoader.getResource("classpath:promptTemplateExample.yaml");
+    final var promptTemplate =
+        new String(resource.getInputStream().readAllBytes(), StandardCharsets.UTF_8);
```

Comment:
(Minor)

This looks a little ugly, have you checked the `@Value` annotation and resource handling?

```java
    @Value("classpath:myfile.txt")
    private Resource resource;

    @GetMapping("/localPromptTemplate")
    Object foo() {
        String content = Files.readString(resource.getFile().toPath(), UTF_8);
```